<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>582</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>24</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 170, 255);
font: 24pt &quot;Arial&quot;;
border-radius:10px;
padding-top: 20px;
padding-bottom: 20px;
padding-right: 5px;
padding-left: 5px;</string>
            </property>
            <property name="text">
             <string>Task Planer</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCalendarWidget" name="calendarWidget"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="newTaskInput">
                  <property name="styleSheet">
                   <string notr="true">QLineEdit{
	border: 2px solid rgb(170, 170, 255);
	font: 14pt &quot;Arial&quot;;
	border-radius:10px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-right: 5px;
	padding-left: 5px;
}
QLineEdit:focus{
	border: 2px solid rgb(0, 170, 255);
	font: 14pt &quot;Arial&quot;;
	border-radius:10px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-right: 5px;
	padding-left: 5px;
}</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="addNewBtn">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
background-color: rgb(170, 170, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius:10px;
padding-top: 8px;
padding-bottom: 8px;
padding-right: 5px;
padding-left: 5px;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	font: 14pt &quot;MS Shell Dlg 2&quot;;
	border-radius:10px;
	padding-top: 8px;
	padding-bottom: 8px;
	padding-right: 5px;
	padding-left: 5px;
}</string>
                  </property>
                  <property name="text">
                   <string>Add New</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QListWidget" name="taskList">
                  <property name="styleSheet">
                   <string notr="true">QListWidget{
	border: 2px solid rgb(170, 170, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QListWidget:focus{
	border: 2px solid rgb(0, 170, 255);
}</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="saveBtn">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
background-color: rgb(170, 170, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius:10px;
padding-top: 8px;
padding-bottom: 8px;
padding-right: 5px;
padding-left: 5px;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	font: 14pt &quot;MS Shell Dlg 2&quot;;
	border-radius:10px;
	padding-top: 8px;
	padding-bottom: 8px;
	padding-right: 5px;
	padding-left: 5px;
}</string>
                  </property>
                  <property name="text">
                   <string>Save Changes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
